FreeBSD 11.1 "make" installation step can't find uuid/uuid.h

Installing ossp-uuid doesn't help.

However, it compiles and works if `/usr/include/uuid.h` is copied to `/usr/include/uuid/uuid.h`.

Thank you for the followup. It’s the current master. Here’s what I saw.

[ 10%] Building CXX object CMakeFiles/libgerbera.dir/src/config/config_generator.cc.o
/root/gerbera/src/config/config_generator.cc:32:10: fatal error: 'uuid/uuid.h'
      file not found
#include <uuid/uuid.h>
         ^~~~~~~~~~~~~
1 error generated.
*** Error code 1

Stop.

And if I: # mkdir /usr/include/uuid && ln -s /usr/include/uuid.h /usr/include/uuid/
It compiles, installs, and runs.


> On Jun 3, 2018, at 1:30 PM, Ian Whyman <notifications@github.com> wrote:
> 
> Which version/tag are you trying to build?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/gerbera/gerbera/issues/310#issuecomment-394177778>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AmCDxuUl9MIqL1MTxbggZtgspa7EpbErks5t5B0-gaJpZM4UYFqU>.
>

--- a/src/config/config_generator.cc
+++ b/src/config/config_generator.cc
@@ -29,7 +29,11 @@ Gerbera - https://gerbera.io/
 #include <common.h>
 #include <tools.h>
 #include <metadata_handler.h>
+#ifdef BSD_NATIVE_UUID
+#include <uuid.h>
+#else
 #include <uuid/uuid.h>
+#endif
 #include "config_generator.h"
 
 using namespace zmm;

